src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+            'Create tag'                          : 'Créer une étiquette',
+            'Tag created'                         : 'Étiquette crée',
+            'Creating tag failed'                 : 'L’étiquette n’a pas pu être effacé',
+            'Edit tag'                            : 'Modifier l’étiquette',
+            'Tag saved'                           : 'Étiquette sauvée',
+            'Saving tag failed'                   : 'L’étiquette n’a pas pu être sauvé',
+            'Delete tag'                          : 'Effacer l’étiquette',
+            'Do you want to delete the tag'       : 'Êtes vous sur de vouloir effacer l’étiquette ?',
+            'Tag deleted'                         : 'Étiquette effacée',
+            'Deleting tag failed'                 : 'L’étiquette n’a pas pu être effacée',
+            'Tag restored'                        : 'L’étiquette restaurée',
+            'Restoring tag failed'                : 'L’étiquette n’a pas pu être restaurée',
+            'Create password'                     : 'Créer un mot de passe',
+            'Password created'                    : 'Mot de passe créé',
+            'Creating password failed'            : 'Le mot de passe n’a pas pu être créé',
+            'Edit password'                       : 'Modifier le mot de passe',
+            'Password saved'                      : 'Mot de passe sauvé',
+            'Saving password failed'              : 'Le mot de passe n’a pas pu être sauvé',
+            'Password moved'                      : 'Mot de passe déplacé',
+            'Moving password failed'              : 'Le mot de passe n’a pas pu être déplacé',
+            'Delete password'                     : 'Effacer le mot de passe',
+            'Do you want to delete the password'  : 'Êtes vous sur de vouloir effacer le mot de passe ?',
+            'Password deleted'                    : 'Mot de passe effacé',
+            'Password restored'                   : 'Mot de passe restauré',
+            'Restoring password failed'           : 'Le mot de passe n’a pas pu être restauré',
+            'Open Url'                            : 'Ouvrir le lien',
+            'Copy Url'                            : 'Copier le lien',
+            'Copy User'                           : 'Copier l’utilisateur',
+            'Copy Password'                       : 'Copier le mot de passe',
+            '{element} was copied to clipboard'   : '{element} ont été copiés dans le presse-papier',
+            'Last modified on {date}'             : 'Dernière modification le {date}',
+            'Restore revision'                    : 'Restaurer une version antérieure',
+            'Revision restored'                   : 'Version restaurée',
+            'Restoring revision failed'           : 'Échec à la restauration antérieure',
+            'Do you want to restore the revision?': 'Êtes vous sur de vouloir restaurer une version antérieure ?',
```

Comment:
Should be `Êtes vous sûr de vouloir restaurer une version antérieure ?` (`sur` => `sûr`)